components/playlist/renderer/playlist_render_frame_observer.cc

```diff
@@ -31,6 +33,9 @@ void PlaylistRenderFrameObserver::RunScriptsAtDocumentStart() {
   const auto current_origin =
       render_frame()->GetWebFrame()->GetSecurityOrigin();
   if (base::ranges::any_of(*origins, [&current_origin](const auto& origin) {
+        if (base::FeatureList::IsEnabled(features::kCompareOnlyHostForTesting))
```

Comment:
Is this added due to port number?

Reply:
@goodov , I'm not sure how to use this to drop port number from  "https://www.youtube.com:51192" `WebSecurityOrigin`. 

Currently, we're using host_resolver() for routing 'youtube.com' request to our embedded server. But the embedded server's port number is attached to the `WebSecurityOrigin`. Could you give me some advices?

Reply:
just open `GURL("https://www.youtube.com/");`, no need to resolve a url via embedded server.

Reply:
Aha! Thanks! Let me try again.